server/admin_management/api/app/utils/utils.py

```diff
@@ -28,3 +29,10 @@ def remove_app_env_suffix(name: str):
         if name.endswith(suffix):
             return name[: -len(suffix)]
     return name
+
+
+def is_success_response(response: Response):
+    SUCCESS_LIST = [
+        HTTPStatus.OK, HTTPStatus.CREATED, HTTPStatus.ACCEPTED, HTTPStatus.NO_CONTENT
+    ]
```

Comment:
I prefer to have this in the constants file, but this is fine too

Reply:
Thanks. This I don't feel it will be used by other, for now just here.